core/src/avm2/globals/flash/globalization/DateTimeFormatter.as

```diff
@@ -0,0 +1,101 @@
+package flash.globalization {
+    import __ruffle__.stub_constructor;
+    import __ruffle__.stub_getter;
+    import __ruffle__.stub_method;
+    import flash.globalization.LastOperationStatus;
```

Comment:
This import is unnecessary

Reply:
It's used on line 32.

Reply:
If anything, I was wondering whether the string literal default parameter values should be their corresponding named constants ("enums", like this one) instead. But the AS3 docs also say string literals there, so...

Reply:
I tried that already and the AS3 compiler doesn't allow it.